### ISSUE TITLE
editing locks crashes the map editor

### ISSUE THREAD

**Issue by JAsmine-ja2** [ja2-stracciatella/ja2-stracciatella#668]
![clipboard01](https://user-images.githubusercontent.com/17213795/34613476-3db34236-f22e-11e7-912f-415fcb50ebae.jpg)

Trying to create a new lock or to edit an existing lock on a map crashes the map editor. Even trying to cancel the "lock-editor" crashes the map editor.

**Comment by Manuel-K** [comment]
I can create new locks. Tested on the lower door of the large building next to the fence on C13.

**Comment by Manuel-K** [comment]
![lock](https://user-images.githubusercontent.com/775439/34643347-c3eeaf88-f31a-11e7-8f8c-db1cbcce819a.png)

**Comment by JAsmine-ja2** [comment]
![edit](https://user-images.githubusercontent.com/17213795/34647834-42c1c75a-f38e-11e7-9007-f3acc86cc23f.jpg)
Can you safely click on "Okay" and "Cancel" - because that crashes my editor.
Do we use the the same editor.slf?

**Comment by Manuel-K** [comment]
>  Can you safely click on "Okay" and "Cancel" - because that crashes my editor.

Yes. I can even see the little lock symbol next to the door after I've added a new lock. If I edit the same door a second time, I can see changes that I've made during the first edit.

> Do we use the the same editor.slf?

I'm using the GOG version. 
```
md5sum:  95ddea8938316e7b2058a7225f2ebd62
sha1sum: 02dd2ac822623dd3caa416642a627b60896159e0
```

**Comment by JAsmine-ja2** [comment]
I don't own the GOG version. I just have physical copies of JA2. This is my editor.slf:
[editor.zip](https://github.com/ja2-stracciatella/ja2-stracciatella/files/1609943/editor.zip)

**Comment by chmallon** [comment]
@JAsmine-ja2: This is identical to Manuel-K's editor.slf.
